### DISCUSSION TITLE
Transparency not working?

### DISCUSSION THREAD

**Discussion by vzchxiao** [mozilla/hubs#5384] [Spoke Questions] [answered]
Hi there,
I created translucent materials in Blender using the Alpha channel, first by trying the slider, then by trying a gray image. Both resulted in translucent materials in blender, but when I import it into Spoke, it becomes opaque. Am I doing something wrong?
<img width="611" alt="Screen Shot 2022-04-25 at 5 06 33 PM" src="https://user-images.githubusercontent.com/104340745/165024362-5de719af-0be2-4710-af12-40f99325a9be.png">
<img width="1792" alt="Screen Shot 2022-04-25 at 5 02 31 PM" src="https://user-images.githubusercontent.com/104340745/165024367-738080c3-1c87-42aa-8dd3-8fa1cfca0213.png">
<img width="1792" alt="Screen Shot 2022-04-25 at 5 02 11 PM" src="https://user-images.githubusercontent.com/104340745/165024372-83529d8c-a109-44c2-9aea-e63563e0596a.png">

**Comment by matthewbcool** [comment]
Have you tried using Alpha Clip instead?

There is an excellent discussion around creating transparent materials in the #art channel in the hubs discord: https://discord.gg/sBMqSjCndj

**Comment by Exairnous** [comment, marked as answer]
It looks like you're using an older version of Blender, and you're probably using its version of the glTF exporter.  In previous versions of the glTF exporter there were a couple of bugs related to alpha that have since been fixed ([glTF2.0 exporter does not export material alpha](https://developer.blender.org/T78754), which was fixed in Blender 2.90, and [unreported] [glTF exporter: fix export when texture is used only for alpha](https://developer.blender.org/rBA5f2cb885abb9), which was fixed in Blender 2.92).

I can confirm that my copy of Blender 2.83 also fails to export alpha, the same as in your screenshots, while 2.93 works fine.

I think there are three options to fix this:
1. Update to a later version of Blender.
2. Download a later version of the exporter (it may need some modification to be compatible with your Blender version) and use that instead.
3. Manually apply the fixes (and there could be more than the two I've identified) to your current version of the exporter.

I hope this helps :)

**Reply by vzchxiao** [reply]
Oh my gee thanks! I updated Blender and it's fixed now
